api/proto/teleport/clusterconfig/v1/access_graph_settings.proto

```diff
@@ -32,12 +32,32 @@ message AccessGraphSettings {
   teleport.header.v1.Metadata metadata = 4;
   // spec is the spec of the resource.
   AccessGraphSettingsSpec spec = 5;
+  // status is the status of the configured access graph service
+  AccessGraphSettingsStatus status = 6;
+}
+
+// AccessGraphSettingsStatus is the status of the configured access graph service
+message AccessGraphSettingsStatus {
+  // initial_sync_complete indicates if the access graph service has sent resources to access graph at least once.
+  bool initial_sync_complete = 1;
 }
 
 // AccessGraphSettingsSpec is the spec for the Access Graph service configuration settings.
 message AccessGraphSettingsSpec {
   // secrets_scan_config is used to configure the parameters for the secrets scanning functionality.
   AccessGraphSecretsScanConfig secrets_scan_config = 1;
+  // demo_mode is used to determine if some features should be available for use in a demo capacity
+  AccessGraphDemoMode demo_mode = 2;
+}
+
+// AccessGraphDemoMode is used to determine if some features should be available for use in a demo capacity
+enum AccessGraphDemoMode {
```

Comment:
It's an enum so will be marshalled into a number in json/yaml. I think the latest consensus is to use string instead if it's going to be stored anywhere.

Reply:
this is just following the pattern for the same type of `enabled/disabled` pattern for the scan config below. really, it just represents a bool. do you feel strongly to changing _this_ field to be different than the existing `AccessGraphSecretsScanConfig` and if so, are you suggesting storing "true/false" as strings? let me know, i dont mind either way

Reply:
I meant

```
string demo_mode = 2; // "enabled"/"disabled"
```

Or just replace it with a bool if it's really an on/off switch.

I haven't noticed we are already using the AccessGraphSecretsScanConfig enum below, so  it's probably ok to leave as is in this case.